app/Actions/Place/PlaceCollectionResponse.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Hedonist\Actions\Place;
+
+use Hedonist\Entities\Place\Place;
+use Illuminate\Database\Eloquent\Collection;
+
+class PlaceCollectionResponse
+{
+    private $id;
+    private $creatorEmail;
+    private $category;
+    private $city;
+    private $longitude;
+    private $latitude;
+    private $zip;
+    private $address;
+
+    public function __construct(Collection $place)
```

Comment:
how does it work? You receive Collection and retrieve from it properties of collection item, I think there is a mistake